pyproject.toml

```diff
@@ -45,6 +45,7 @@ awscli = [
 boto3 = [
     "boto3 >= 1.38.23, < 1.38.25",
 ]
+httpx = ["httpx"]
```

Comment:
Could you break this into three lines, following the examples set above?

It would be good to add version constraints, too. Lower constraint, if you use features that were introduced at some point. Upper constraint to prevent breaking changes to affect users.